dev/manual_tests/lib/density.dart

```diff
@@ -629,8 +629,9 @@ class _MyHomePageState extends State<MyHomePage> {
               data: Theme.of(context).copyWith(visualDensity: _model.density),
               child: Directionality(
                 textDirection: _model.rtl ? TextDirection.rtl : TextDirection.ltr,
-                child: MediaQuery(
-                  data: MediaQuery.of(context).copyWith(textScaleFactor: _model.size),
+                child: MediaQuery.withClampedTextScaling(
```

Comment:
This seems like a obscure way to create a linear text scalar

should this do a copyWith `TextScaler.linear`?